DQMOffline/Ecal/plugins/EcalPileUpDepMonitor.cc

```diff
@@ -340,7 +340,7 @@ void EcalPileUpDepMonitor::analyze(const edm::Event &e, const edm::EventSetup &)
     const EcalRecHitCollection *eeRecHits = RecHitsEE.product();
 
     reco::BasicCluster const &seedCluster(*itSC->seed());
-    std::vector<float> cov = EcalClusterTools::localCovariances(seedCluster, eeRecHits, p_topology);
+    std::array<float, 3> cov = EcalClusterTools::localCovariances(seedCluster, eeRecHits, p_topology);
```

Comment:
to capture the return it would be better to use auto everywhere
```suggestion
    const auto& cov = EcalClusterTools::localCovariances(seedCluster, eeRecHits, p_topology);
```
or 
```suggestion
    const auto cov = EcalClusterTools::localCovariances(seedCluster, eeRecHits, p_topology);
```

I'm not sure which is better honestly, @VinInn what do you think?